setup.cfg

```
@@ -83,6 +83,7 @@ conch =
     cryptography >= 2.6
     appdirs >= 1.4.0
     bcrypt >= 3.0.0
+    PyNaCl
```

Maybe add a separate target.

Most users will have a modern cryptography and nacl is not needd

```suggestion
conch-nacl:
    %(conch)s
    PyNaCl
```

---

Seems reasonable.  Done (not quite via your suggestion, but similar).